[CI] TSDB Mixed cluster tests failing

These 5 tests have failed on multiple branches (8.6, 8.5, 8.4, 8.3, 8.2), the tests are skipped on many branches but not those 

https://github.com/elastic/elasticsearch/blob/9f2d38856d3f689344fe768e10a59a5eb91438c4/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/25_id_generation.yml#L4

MixedClusterClientYamlTestSuiteIT » test {p0=tsdb/25_id_generation/create operation on top of old document MixedClusterClientYamlTestSuiteIT » test {p0=tsdb/30_snapshot/Create a snapshot and then restore it}
MixedClusterClientYamlTestSuiteIT » test {p0=tsdb/25_id_generation/generates a consistent id}
MixedClusterClientYamlTestSuiteIT » test {p0=tsdb/25_id_generation/index a new document on top of an old one over bulk}
MixedClusterClientYamlTestSuiteIT » test {p0=tsdb/25_id_generation/delete over _bulk}


### CI Link

https://gradle-enterprise.elastic.co/s/ipuwzm5n5spp6/tests/:qa:mixed-cluster:v8.5.0%23mixedClusterTest/org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT/test%20%7Bp0=tsdb%2F30_snapshot%2FCreate%20a%20snapshot%20and%20then%20restore%20it%7D

https://gradle-enterprise.elastic.co/s/ipuwzm5n5spp6/tests/:qa:mixed-cluster:v8.5.0%23mixedClusterTest/org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT/test%20%7Bp0=tsdb%2F25_id_generation%2Fdelete%20over%20_bulk%7D

https://gradle-enterprise.elastic.co/s/ipuwzm5n5spp6/tests/:qa:mixed-cluster:v8.5.0%23mixedClusterTest/org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT/test%20%7Bp0=tsdb%2F25_id_generation%2Fgenerates%20a%20consistent%20id%7D

https://gradle-enterprise.elastic.co/s/ipuwzm5n5spp6/tests/:qa:mixed-cluster:v8.5.0%23mixedClusterTest/org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT/test%20%7Bp0=tsdb%2F25_id_generation%2Fcreate%20operation%20on%20top%20of%20old%20document%20fails%7D

https://gradle-enterprise.elastic.co/s/ipuwzm5n5spp6/tests/:qa:mixed-cluster:v8.5.0%23mixedClusterTest/org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT/test%20%7Bp0=tsdb%2F25_id_generation%2Findex%20a%20new%20document%20on%20top%20of%20an%20old%20one%20over%20bulk%7D

### Repro line

./gradlew ':qa:mixed-cluster:v8.5.0#mixedClusterTest' -Dtests.class="org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT" -Dtests.method="test {p0=tsdb/30_snapshot/Create a snapshot and then restore it}" -Dtests.seed=79E8550FE6BF2740 -Dtests.bwc=true -Dtests.locale=he-IL -Dtests.timezone=Pacific/Chatham -Druntime.java=21


./gradlew ':qa:mixed-cluster:v8.5.0#mixedClusterTest' -Dtests.class="org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT" -Dtests.method="test {p0=tsdb/25_id_generation/delete over _bulk}" -Dtests.seed=79E8550FE6BF2740 -Dtests.bwc=true -Dtests.locale=he-IL -Dtests.timezone=Pacific/Chatham -Druntime.java=21	

./gradlew ':qa:mixed-cluster:v8.5.0#mixedClusterTest' -Dtests.class="org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT" -Dtests.method="test {p0=tsdb/25_id_generation/delete over _bulk}" -Dtests.seed=79E8550FE6BF2740 -Dtests.bwc=true -Dtests.locale=he-IL -Dtests.timezone=Pacific/Chatham -Druntime.java=21	

./gradlew ':qa:mixed-cluster:v8.5.0#mixedClusterTest' -Dtests.class="org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT" -Dtests.method="test {p0=tsdb/25_id_generation/generates a consistent id}" -Dtests.seed=79E8550FE6BF2740 -Dtests.bwc=true -Dtests.locale=he-IL -Dtests.timezone=Pacific/Chatham -Druntime.java=21	


./gradlew ':qa:mixed-cluster:v8.5.0#mixedClusterTest' -Dtests.class="org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT" -Dtests.method="test {p0=tsdb/25_id_generation/create operation on top of old document fails}" -Dtests.seed=79E8550FE6BF2740 -Dtests.bwc=true -Dtests.locale=he-IL -Dtests.timezone=Pacific/Chatham -Druntime.java=21	


./gradlew ':qa:mixed-cluster:v8.5.0#mixedClusterTest' -Dtests.class="org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT" -Dtests.method="test {p0=tsdb/25_id_generation/index a new document on top of an old one over bulk}" -Dtests.seed=79E8550FE6BF2740 -Dtests.bwc=true -Dtests.locale=he-IL -Dtests.timezone=Pacific/Chatham -Druntime.java=21	

### Does it reproduce?

Didn't try

### Applicable branches

8.6, 8.5, 8.4, 8.3, 8.2

### Failure history

https://gradle-enterprise.elastic.co/scans/tests?search.timeZoneId=Europe%2FLondon&tests.container=org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT&tests.test=test%20%7Bp0%3Dtsdb%2F25_id_generation%2Fgenerates%20a%20consistent%20id%7D

### Failure excerpt

```
java.lang.AssertionError: Failure at [tsdb/25_id_generation:79]: |  
-- | --
  | Expected: "cZZNs7B9sSWsyrL5AAABeRnS7fM" |  
  | but: was "cZZNs4NdV58ePSPIAAABeRnS7fM"
```

```

java.lang.AssertionError: Failure at [tsdb/25_id_generation:397]: |  
-- | --
  | Expected: <true> |  
  | but: was <false>
```

```
java.lang.AssertionError: Failure at [tsdb/25_id_generation:218]: |  
-- | --
  | Expected: "cn4excfoxSs_KdA5AAABeRnR_mY" |  
  | but: was "cn4exTOUtxytuLkQAAABeRnR_mY"
```

--- a/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/25_id_generation.yml
+++ b/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/25_id_generation.yml
@@ -1,8 +1,8 @@
 ---
 setup:
   - skip:
-      version: " - 8.1.99,8.7.00 - 8.12.99"
-      reason: _tsid hashing introduced in 8.13
+      version: " - 8.1.99,8.7.00 - 8.9.99"
+      reason: "tsdb indexing changed in 8.2.0, synthetic source shows up in the mapping in 8.10 and on, may trigger assert failures in mixed cluster tests"
 
   - do:
       indices.create:

--- a/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/25_id_generation.yml
+++ b/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/25_id_generation.yml
@@ -1,8 +1,8 @@
 ---
 setup:
   - skip:
-      version: " - 8.1.99,8.7.00 - 8.9.99"
-      reason: "tsdb indexing changed in 8.2.0, synthetic source shows up in the mapping in 8.10 and on, may trigger assert failures in mixed cluster tests"
+      version: "- 8.12.99"
+      reason: _tsid hashing introduced in 8.13
 
   - do:
       indices.create:

--- a/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/30_snapshot.yml
+++ b/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/30_snapshot.yml
@@ -1,7 +1,7 @@
 ---
 setup:
   - skip:
-      version: "8.7.00 - 8.12.99"
+      version: " - 8.12.99"
       reason: _tsid hashing introduced in 8.13
 
   - do:

